addon/components/fixtable-grid.js

```diff
@@ -462,5 +466,18 @@ export default Ember.Component.extend({
     if (fixtable) {
       fixtable.setDimensions();
     }
+  },
+
+  didUpdateAttrs() {
+    this._super(...arguments);
+
+    // trigger content reload when externalFilters change
+    let oldExternalFilters = this.get('_previousExternalFilters');
+    let newExternalFilters = _.cloneDeep(this.get('externalFilters'));
+    if (oldExternalFilters && !_.isEqual(oldExternalFilters, newExternalFilters)) {
```

Comment:
Do we need lodash when we can do `Ember.copy` and `Ember.isEqual`?  Will these meet your needs here instead of lodash?

Reply:
`Ember.copy` is shallow and we want to ensure objects at any arbitrary depth aren't copied by reference to ensure we actually detect changes. Similarly, `Ember.isEqual` doesn't do the deep comparison we need.

Reply:
![screenshot 2017-10-10 11 02 31](https://user-images.githubusercontent.com/1676335/31397052-8fea9bbc-adaa-11e7-8716-11dae8027108.png)

Reply:
Though you have a point w/ isEqual.  I'm just hesitant to bring in another dependency that can get out of whack w/ web-directory ( or other parent apps ) if we can avoid it.

Reply:
Fair enough about the deep copy, but not sure what to do about isEqual. Do you think it's any better to just bring in the lodash.isequal package as a dependency?

Reply:
Another option I thought of would be to `JSON.stringify` the external filters and compare the strings. I assume it's less performant, but considering it's only going to run when the consumer marks the property as changed I'm not too concerned about a minor difference there. The bigger issue would be reordering of array items being picked up as a change. But we'd remove the lodash dependency. Thoughts?

Reply:
Yeah don't use JSON.stringify. Given the choices, I think lodash is the best, but the architecture here of needing to deep compare two objects is a bit dirty. Any other ideas? Is the developer is explicitly changing the contents of the object/array? Can they MarkModified()?

Reply:
@rbrcurtis The reason I need to deep compare is that I want to be totally agnostic about the structure of externalFilters, which means I can't set up actual observers. And the way we're triggering this hook in the first place is for the controller in the consuming app to mark the property as changed, but the issue is this hook gets triggered whenever *any* attr on the component changes, and we're not given any info as to which attr was updated.